### PR TITLE
Fix some CoffeeScript to ES6 porting bugs + nits

### PR DESCRIPTION
**Pull request by elisee**
Hey! So I took a little look around in the app and some of its code. Nice to see you switched to ES6 (just a step away from TypeScript, maybe! ^_^). Very excited to see it all mature into a fully-fledged itch.io app :)

Anyway, we ported of loooot of CoffeeScript to TypeScript in Superpowers a few months ago so I remembered some of the usual gotchas and went looking for them.
- Found a few template strings that needed to be updated (`"#{goodOldCoffeeScript}"` to ``${shinyES6}``)
- There was a `for (... in ...)` that should be `for (... of ...)` in `format.es6` but it looks like this code isn't used at the moment.
- Added a few semicolons

Hope you dig it! Let me know if there are any changes you don't like and I'll happily fix up the PR.

### CODE REVIEW

File: app/chrome/helpers/format.es6 [hunk 1]
```diff
@@ -8,9 +8,9 @@ let thresholds = [
 ];
 
 function format_bytes (bytes) {
-    for (var [label, min] in thresholds) {
+    for (let [label, min] of thresholds) {
       if (bytes >= min) {
-        return `${_str.numberFormat(bytes / min)}${label}`;
+        return `${_str.numberFormat(bytes / min)} ${label}`;
```
Comment (elisee) [inline app/chrome/helpers/format.es6:13]: I added a space here to match with https://github.com/elisee/itchio-app/blob/87ea5ce80ccfde9ae5adf95e91a82a4525cc279a/app/chrome/helpers/format.es6#L17 below but maybe this was on purpose. Let me know if you don't like the change.

**Comment by fasterthanlime** [comment]
:star2: Hey @elisee, nice to see you here :)

Good catches, thanks! :shipit: 

The only thing is... I wish you found out about [the specs branch](https://github.com/itchio/itchio-app/tree/specs) though - I thought I'd have time to finish it yesterday and merge it into master, but I have a few things to finish up first.

In the specs branch, we're switching to :trophy: [JavaScript standard style](http://standardjs.com/) — I'll let you read the rules, but basically our `.travis.yml` runs a linter and it would've checked for things like semi-colons (actually forbidden in this coding style!).

Let me know if you want to rebase this on `specs` yourself or if you'd rather have me do it, it's not a lot of changes anyway. I didn't expect a PR to drop this morning! :hamster:

**Comment by fasterthanlime** [comment]
(I'm actually rebasing it now, don't bother!)

**Comment by fasterthanlime** [comment]
Cherry-picked as f0872107923b3ec599153d2b1102f7c12c294215

**Comment by elisee** [comment]
Cool :). Should have checked for other branches I guess!

Noticed you kept the `for (... in ...)` at https://github.com/itchio/itchio-app/blob/c41c67f7c0287eee66f5382b991bc10945c57d3d/app/chrome/helpers/format.es6#L11, was my fix wrong or just an oversight on your part?

Unless I'm mistaken, `for (... in ...)` over an array will result in iterating over the object's keys (so the array indices, as strings, `"0", "1", ...`), which is not what the code is trying to achieve AFAICT.

**Comment by fasterthanlime** [comment]
Ah, I missed that one. My bad!

**Pull request closed without merging**